modules/worker-utils/src/lib/worker-farm/worker-farm.js

```diff
@@ -41,12 +41,20 @@ export default class WorkerFarm {
 
   getWorkerPool({name, source, url}) {
     let workerPool = this.workerPools.get(name);
+
+    const maxConcurrency = isMobile ? this.props.maxMobileConcurrency : this.props.maxConcurrency;
+
+    if (workerPool && workerPool.maxConcurrency !== maxConcurrency) {
```

Comment:
Why?

Reply:
This is code that I still need to revisit, because it worked only before the worker logic was refactored. the goal here is to be able to change the `maxConcurrency` in runtime. I found it useful to be able to quickly test different concurrency configurations using the gui.

Reply:
Perhaps we could make `WorkerFarm.setProps({maxConcurrency})` call workerPool.setProps({maxConcurrency}) on all  workerPools?

If so, WorkerPools may need some logic to destroy returned workers if the maxConcurrency has been reduced.

Reply:
Yes this might be the missing piece. I will work on that one later on. Thanks!

Reply:
Hi @ibgreen!
I revisited this code now and it does actually work. Earlier I just forgot to alias the new worker-utils module.
The `getWorkerFarm` method already calls `setProps`, so eventually `maxConcurrency` propagates from the load options down to the worker pool. Essentially with this code, every time a loader is passed a `maxConcurrency` that is different from the existing value, the worker threads are destroyed and then recreated with the updated number.
Do you think this solution can work or do you foresee an issue?

Reply:
I don't see it. 
- The workerFarm just passes the props when it creates a new workerPool, not during workerFarm.setProps. 
- I think it could work if workerFarm iterated over all its current workerPools in `setProps`.